dohq_artifactory/admin.py

```diff
@@ -707,6 +707,7 @@ def __init__(
         packageType=Repository.GENERIC,
         dockerApiVersion=Repository.V1,
         repoLayoutRef="maven-2-default",
+        maxUniqueTags=0,
```

Comment:
@allburov 
I propose to start using Python snake case convention for new attributes
```suggestion
        max_unique_tags=0,
```

later we can think how to keep backwards compatibility for already existing attributes

Reply:
@beliaev-maksim , done